client/my-sites/exporter/export-card/index.jsx

```diff
@@ -91,15 +94,16 @@ const mapStateToProps = ( state, { siteId } ) => ( {
 
 const trackExportClick = ( scope = 'all' ) => recordTracksEvent( 'calypso_export_start_button_click', { scope } );
 
-const mapDispatchToProps = ( dispatch, { siteId } ) => ( {
+const mapDispatchToProps = ( dispatch, ownProps ) => ( {
```

Comment:
There aren't any hard guidelines in our dev docs but you'll see this pattern throughout the codebase. I personally find that it keeps the data requirements very clear. At a glance we can tell that we're only going to be using props.siteId and props.siteType, even if props contains a number of other attributes.